bin/install_cocoapods

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+# Start by restoring specs repo and pod cache:
+# - The specs repo cache holds all of the Podspec files. This avoids having download them all from the CDN.
+# - The pod cache holds the downloaded Pod source files. This avoids having to check them out again.
+restore_cache "$BUILDKITE_PIPELINE_SLUG-specs-repos"
+restore_cache "$BUILDKITE_PIPELINE_SLUG-global-pod-cache"
```

Comment:
I wonder if extracting these cache keys definitions in a shared file to `source` here and in the `cache_cocoapods` script would be useful or make the code harder to follow 🤔

Reply:
Yeah, I thought about DRY-ing this, but given the fact that it should never change, I feel like it's ok – the worst case if we break it is that builds take longer, which isn't the end of the world IMHO? And I think we'd notice it before long, so I went for the less complex route 🤷‍♂️

Reply:
Yeah. Fine by me. If we discover this needs to change (which I can't foresee ATM) we can always refine 👍